src/core/properties/fileproperty.cpp

```diff
@@ -261,6 +262,12 @@ void FileProperty::requestFile() {
     }
 }
 
+void FileProperty::updateWidgets() const {

```

Comment:
this is quite confusing the the same function in the base class Property::updateWidgets() which is protected though. Might be more clear to make that one virtual and overriding it...

Reply:
ah, was not aware of that function.

Reply:
Probably, you could already do for(auto w : p.getWidgets()) w.updateFromProperty();
which would be the same more or less. So there is not much reason for hiding it...